server/middleware/forceHTTPS.js

```diff
@@ -0,0 +1,12 @@
+function forceHTTPS(req, res, next) {
+
+    if (process.env.FORCE_HTTPS === 'true' && !req.secure) {
+       return res.redirect("https://" + req.headers.host + req.url);
+    }
+
+    next();
+}
+
+module.exports = {
+    forceHTTPS
+}
```

Comment:
nit: let's add an empty newline at the end of the file to appease the `-` in the diff here^